_posts/2016-06-10-voxpupuli-opensource-speedtrain.md

```diff
@@ -43,7 +43,7 @@ What is Voxpupuli
 
 > We are a collective of Puppet module, tooling and documentation authors all working together to ensure continued development on the code we maintain.
 
-Some of my favourite words are in this sentence:
+Some of my favorite words are in this sentence:
```

Comment:
Personally I prefer British English instead of American English :)

Reply:
my bad, never trust a yank with the British English language.  Please keep British English.